pages/user.js

```diff
@@ -1,10 +1,22 @@
+/* eslint-disable no-unused-vars */
```

Comment:
remove this disable

Reply:
This still has to be removed, you can not have unused variables in your files